### PR TITLE
Move all assays to module

### PR DESCRIPTION
**Pull request by rctauber**
All children of `assay` are now in the assays module.
Added additional verify step to ensure there are no asserted children of `assay` in the edit file.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -161,21 +162,21 @@ verify: verify-edit verify-merged
...
+verify-merged: build/obi_merged.owl $(MERGED_VIOLATION_QUERIES)
 	$(ROBOT) merge \
 	--input $< \
 	verify \
 	--output-dir build \
 	--queries $(MERGED_VIOLATION_QUERIES)
 
 .PHONY: test
-test: verify
+test: | build/robot.jar verify
```
Comment (jamesaoverton) [inline Makefile]: Should remain `test: verify`

File: Makefile [hunk 2]
```diff
@@ -161,21 +162,21 @@ verify: verify-edit verify-merged
 
 # Run validation queries on obi-edit and exit on error.
 .PHONY: verify-edit
-verify-edit: src/ontology/obi-edit.owl  $(EDIT_VIOLATION_QUERIES) | build
+verify-edit: src/ontology/obi-edit.owl $(EDIT_VIOLATION_QUERIES)
```
Comment (jamesaoverton) [inline Makefile]: Should be `verify-edit: src/ontology/obi-edit.owl $(EDIT_VIOLATION_QUERIES) | build/robot.jar`

File: Makefile (hunk 1 above)
Comment (rctauber) [reply Makefile]: Ok - for some reason the build/robot.jar wasn't being found so I had to throw that in there. Should I move it to -
```
verify: | build/robot.jar verify-edit verify-merged
```
?

File: Makefile [hunk 3]
```diff
@@ -161,21 +162,21 @@ verify: verify-edit verify-merged
...
 	$(ROBOT) verify --input $< --output-dir build \
 	--queries $(EDIT_VIOLATION_QUERIES)
 
 # Run validation queries on obi_merged and exit on error.
 .PHONY: verify-merged
-verify-merged: build/obi_merged.owl $(MERGED_VIOLATION_QUERIES) | build
+verify-merged: build/obi_merged.owl $(MERGED_VIOLATION_QUERIES)
```
Comment (jamesaoverton) [inline Makefile]: Should be `verify-merged: build/obi_merged.owl $(MERGED_VIOLATION_QUERIES) | build/robot.jar`

**Comment by jamesaoverton** [comment]
This PR is using a new ROBOT feature that will be in the next release. That's the only reason that the test have failed for the last couple of commits. @rctauber and I have run the tests locally with development versions of ROBOT and all the tests pass.

**Pull request merged**